test/uk/gov/hmrc/gform/sharedmodel/JsonParseTestFormat.scala

```diff
@@ -101,4 +98,28 @@ class JsonParseTestFormat extends Spec {
 
     }
   }
+
+  "A component with a valid display width" should "parse correctly" in {
+
+    val displayWidthOptions = Table(
+      // format: off
+      ("displayWidth", "expected"),
+      (DisplayWidthAttribute.XS.toString.toLowerCase,  Text(ShortText, Value, DisplayWidthAttribute.XS)),
+      (DisplayWidthAttribute.S.toString.toLowerCase,   Text(ShortText, Value, DisplayWidthAttribute.S)),
+      (DisplayWidthAttribute.M.toString.toLowerCase,   Text(ShortText, Value, DisplayWidthAttribute.M)),
+      (DisplayWidthAttribute.L.toString.toLowerCase,   Text(ShortText, Value, DisplayWidthAttribute.L)),
+      (DisplayWidthAttribute.XL.toString.toLowerCase,  Text(ShortText, Value, DisplayWidthAttribute.XL)),
+      (DisplayWidthAttribute.XXL.toString.toLowerCase, Text(ShortText, Value, DisplayWidthAttribute.XXL))
```

Comment:
Maybe to add test for wrong display width to see default case (`L`)

Reply:
Yes, will do